sequelize/seeders/20180711143600-memeolist-example.js

```diff
@@ -64,6 +75,12 @@ const resolvers = [
       }
     }`,
     responseMapping: '{{ toJSON (convertNeDBIds context.result) }}',
+    publish: JSON.stringify({
+      topic: 'memeCreated',
+      payload: `{
+        "memeAdded": {{ toJSON context.result }},
+      }`
+    }),
```

Comment:
can you also add sample into `memeolist.query.graphql` file so that we don't have to type anything when we open Graphiql?